lib/index.js

```diff
@@ -1006,6 +1017,78 @@ MapboxGeocoder.prototype = {
     return this.options.origin;
   },
 
+  /**
+   * Set the autocomplete option used for geocoding requests
+   * @param {Boolean} value The boolean value to set autocomplete to
+   * @returns 
+   */
+  setAutocomplete: function(value){
+    this.options.autocomplete = value;
+    return this;
+  },
+
+  /**
+   * Get the current autocomplete parameter value used for requests
+   * @returns {Boolean} The autocomplete parameter value
+   */
+  getAutocomplete: function(){
+    return this.options.autocomplete
+  },
+
+  /**
+   * Set the fuzzyMatch option used for approximate matching in geocoding requests
+   * @param {Boolean} value The boolean value to set fuzzyMatch to
+   * @returns 
+   */
+  setFuzzyMatch: function(value){
+    this.options.fuzzyMatch = value;
+    return this;
+  },
+
+  /**
+   * Get the current fuzzyMatch parameter value used for requests
+   * @returns {Boolean} The fuzzyMatch parameter value
+   */
+  getFuzzyMatch: function(){
+    return this.options.fuzzyMatch
+  },
+
+  /**
+   * Set the routing parameter used to ask for routable point metadata in geocoding requests
+   * @param {Boolean} value The boolean value to set routing to
+   * @returns 
+   */
+  setRouting: function(value){
+    this.options.routing = value;
+    return this;
+  },
+
+  /**
+   * Get the current routing parameter value used for requests
+   * @returns {Boolean} The routing parameter value
+   */
+  getRouting: function(){
+    return this.options.routing
+  },
+
+  /**
+   * Set the worldview parameter
+   * @param {String} code The country code representing the worldview (e.g. "us" | "cn" | "jp", "in")
+   * @returns 
+   */
+  setWorldview: function(code){
+    this.options.worldview = code;
+    return this;
+  },
+
+  /**
+   * Get the current worldview parameter value used for requests
+   * @returns {Boolean} The worldview parameter value
```

Comment:
Update worldview response to @returns {String}

Reply:
Good catch! Done